taskcluster/ci/signing/kind.yml

```diff
@@ -20,15 +20,15 @@ job-template:
     description: Sign Fenix
     worker-type:
         by-build-type:
-            (fennec-.+|nightly|android-test-nightly):
+            (nightly|beta|release|android-test-nightly):
                 by-level:
                     '3': signing
                     default: dep-signing
             default: dep-signing
     worker:
         signing-type:
             by-build-type:
-                fennec-(beta|production):
```

Comment:
Last nit: Update `index.by-build-type` down below, with the same kind of changes. My local testing shows the beta signing task isn't indexed anymore.